packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_title_card.dart

```diff
@@ -84,7 +88,7 @@ class KnowledgePanelTitleCard extends StatelessWidget {
       case Evaluation.BAD:
         return RED_COLOR;
       case Evaluation.NEUTRAL:
-        return GREY_COLOR;
+        return PRIMARY_GREY_COLOR;
```

Comment:
Too dark to work on dark theme. If you want different colors for dark/light, create a method for that.

Reply:
Hi @monsieurtanuki ,
I feel that a single color should be used for both modes just like for the rest of the cases.
I've used `PRIMARY_BLUE_COLOR`.

Reply:
> I feel that a single color should be used for both modes just like for the rest of the cases.

That's fair, as long as the color has enough contrast with the background:
* https://www.material.io/design/usability/accessibility.html#color-and-contrast
* https://www.material.io/resources/color/#!/?view.left=1&view.right=0

> I've used PRIMARY_BLUE_COLOR.

Btw the choice of colors is not mine. What do the mock-ups say?